rero_ils/modules/documents/jsonschemas/documents/document-v0.0.1.json

```diff
@@ -1613,6 +1613,10 @@
             "items": {
               "type": "object",
               "properties": {
+                "_text": {
+                  "title": "Calculated text",
```

Comment:
Can you change this such as: normalized version of the provisionActivity or provisionActivity statement, etc.